<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrgComboBoxPopupEx</class>
 <widget class="QWidget" name="OrgComboBoxPopupEx">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabOrg">
      <attribute name="title">
       <string>&amp;Организации</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <widget class="QListView" name="tblOrg"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSearch">
      <attribute name="title">
       <string>&amp;Поиск</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lblName">
         <property name="text">
          <string>&amp;Название содержит</string>
         </property>
         <property name="buddy">
          <cstring>edtName</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="edtName"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblINN">
         <property name="text">
          <string>&amp;ИНН</string>
         </property>
         <property name="buddy">
          <cstring>edtINN</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="edtINN"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblOKATO">
         <property name="text">
          <string>ОКАТО</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0" colspan="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>172</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="11" column="2">
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Apply|QDialogButtonBox::Reset</set>
         </property>
         <property name="centerButtons">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblOGRN">
         <property name="text">
          <string>&amp;ОГРН</string>
         </property>
         <property name="buddy">
          <cstring>edtOGRN</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="edtOGRN"/>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QLineEdit" name="edtOKATO"/>
       </item>
       <item row="10" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="8" column="1" colspan="2">
        <widget class="CRBComboBox" name="cmbHospitalBedProfile"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="lblHospitalBedProfile">
         <property name="text">
          <string>Профиль койки</string>
         </property>
         <property name="buddy">
          <cstring>cmbHospitalBedProfile</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QLineEdit" name="edtTfomsCode"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblTfomsCode">
         <property name="text">
          <string>Код ТФОМС</string>
         </property>
         <property name="buddy">
          <cstring>edtTfomsCode</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QLineEdit" name="edtOKVED"/>
       </item>
       <item row="6" column="1" colspan="2">
        <widget class="QComboBox" name="cmbIsInsurer">
         <item>
          <property name="text">
           <string>не задано</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Нет</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Да</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lblOKVED">
         <property name="text">
          <string>ОКВЭД</string>
         </property>
         <property name="buddy">
          <cstring>edtOKVED</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lblActive">
         <property name="text">
          <string>Активность</string>
         </property>
         <property name="buddy">
          <cstring>cmbActive</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="2">
        <widget class="QComboBox" name="cmbActive">
         <item>
          <property name="text">
           <string>не задано</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Не активна</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Активна</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lblIsInsurer">
         <property name="text">
          <string>Страховая компания</string>
         </property>
         <property name="buddy">
          <cstring>cmbIsInsurer</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="lblIsMedical">
         <property name="text">
          <string>ЛПУ</string>
         </property>
         <property name="buddy">
          <cstring>cmbIsMedical</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="1" colspan="2">
        <widget class="QComboBox" name="cmbIsMedical">
         <item>
          <property name="text">
           <string>не определено</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>поликлиника</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>стационар</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>прочая мед.организация</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>СМП</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>санаторного типа</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tblOrg</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>edtINN</tabstop>
  <tabstop>edtOGRN</tabstop>
  <tabstop>edtOKATO</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
